app/views/docs/graphql.phtml

```diff
@@ -0,0 +1,497 @@
+<p>Appwrite supports multiple protocols for accessing the server, including <a href="/docs/rest">REST</a>, <a href="/docs/graphql">GraphQL</a>, and <a href="/docs/realtime">Realtime</a>.</p>
+
+<p>The GraphQL API allows you to query and mutate any resource type on your Appwrite server through a single endpoint. Every endpoint available via REST is available via a corresponding GraphQL field.</p>
+
+<h2><a href="/docs/graphql#requests" id="requests">Requests</a></h2>
+
+<p>GraphQL requests can be made in multiple ways, as every action executes through the same endpoint. However, all requests have a common structure.</p>
+
+<table class="vertical full args">
+    <thead>
+    <tr>
+        <td style="width: 140px">Name</td>
+        <td style="width: 70px"></td>
+        <td style="width: 120px">Type</td>
+        <td>Description</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td data-title="Name: ">
+            query
+        </td>
+        <td data-title="">
+            <span class="tag red">required</span>
+        </td>
+        <td data-title="Type: ">
+            string
+        </td>
+        <td data-title="Description: ">
+            The GraphQL query to execute.
+        </td>
+    </tr>
+    <tr>
+        <td data-title="Name: ">
+            operationName
+        </td>
+        <td data-title="">
+            <span class="tag">optional</span>
+        </td>
+        <td data-title="Type: ">
+            string
+        </td>
+        <td data-title="Description: ">
+            If the query contains several named operations, controls which one to execute.
+        </td>
+    </tr>
+    <tr>
+        <td data-title="Name: ">
+            variables
+        </td>
+        <td data-title="">
+            <span class="tag">optional</span>
+        </td>
+        <td data-title="Type: ">
+            object
+        </td>
+        <td data-title="Description: ">
+            An object containing variable names and values for the query. Variables are made available to your query with the <code>$</code> prefix.
+        </td>
+    </tr>
+    </tbody>
+</table>
+
+<h3><a href="/docs/graphql#get-requests" id="get-requests">GET Requests</a></h3>
+
+<p>You can execute a GraphQL query via a GET request, passing a <b>query</b> and optionally an <b>operationName</b> and <b>variables</b> as query parameters.</p>
+
+<h3><a href="/docs/graphql#post-requests" id="post-requests">POST Requests</a></h3>
+
+<p>The multiple ways to make a GraphQL POST request first break down by content type.</p>
+
+<h4>JSON</h4>
+
+<p>There are two ways to make requests with the <b>application/json</b> content type. You can send a JSON object containing a query and optionally an operationName and variables, or an array of objects with the same structure.</p>
```

Comment:
I think `query`, `operationName` and `variables` should be bolded like they are in the GET Requests section.

```suggestion
<p>There are two ways to make requests with the <b>application/json</b> content type. You can send a JSON object containing a <b>query</b> and optionally an <b>operationName</b> and <b>variables</b>, or an array of objects with the same structure.</p>
```